app/views/labware/_basic_relative.html.erb

```diff
@@ -1,11 +1,11 @@
 <%# A simple representation of any labware type not currently handled otherwise -%>
 <%# locals: (labware:, open_in_new_window: false) -%>
 <%
-  labware_path = limber_plate_path(labware.uuid) + "#relatives"
+  labware_path = "#{limber_plate_path(labware.uuid)}#relatives"
 %>
 <div class="relative-item">
     <%= labware.purpose.name %> - <%= link_to labware_path, class: 'card-link stretched-link', target: (open_in_new_window ? '_blank' : '_self') do %>
-        <%= labware.barcode.human.present? ? labware.barcode.human : labware.barcode.machine %>
+        <%= labware.barcode.human.presence || labware.barcode.machine %>
```

Comment:
Hm, does `.presence` still return the `labware.barcode.human` object if it exists? Nice if so!